### ISSUE TITLE
avy-mode and ace-jump fonts much too difficult to see

### ISSUE THREAD

**Issue by impaktor** [bbatsov/zenburn-emacs#215]
The current colour-scheme is anything but helpful when using ace-jump mode.

Consider this Python snippet, where I want to navigate to the `False` on the second line.
![ace_zen2](https://cloud.githubusercontent.com/assets/619390/11532607/2b518a32-9905-11e5-886a-b49d51f07c0e.png)
So activate ace-jump:
![ace_zen1](https://cloud.githubusercontent.com/assets/619390/11532600/20052882-9905-11e5-8be9-9ff80754c327.png)
See how the "F" is replaced with a completly different character? Me neither.

We might all like discrete fonts and low constrast colour-schemes, like the line numbers, but sometimes we need to see things that "pop" at us, especially when it's stuff that are only (meant to be) shown for a few seconds.

I think the most sane thing would be to __set a fixed background colour__ and a __contrasting foreground__ colour, it's the best way to make it visible also in various syntax highlighting / font lock schemes.

I have not looked further into which zenburn colours might be useful for this, but gray background is _not_ what I would suggest here, but perhaps rather the inverse.

To demonstrate, this is how clear it can be in other colourschemes:
![avy-goto-char](https://raw.githubusercontent.com/wiki/nloyola/avy/images/avy-goto-char.png)
(from https://github.com/abo-abo/avy)

**Comment by bbatsov** [comment]
As mentioned before - I'm fine with something like this as long as it's conditional.

**Comment by bbatsov** [comment]
@impaktor @manuel-uberti Btw, I failed to mention this before - you're probably not setting `avy-background` to `t` (like I do). I'm guessing the conditional check should be exactly this - stronger backgrounds make sense only when you're not using `avy-background` (which is dimming everything else).

**Comment by bbatsov** [comment]
![image](https://cloud.githubusercontent.com/assets/103882/14065387/df1e1cda-f42f-11e5-96ac-09342a598406.png)

This is how avy looks for me.

**Comment by bbatsov** [comment]
Anyways, it's easy to change those in personal configs, so I'll close this ticket.

**Comment by impaktor** [comment]
> I'm fine with something like this as long as it's conditional.

I currently don't have the time to aquire the needed skills to do that, but I already run my own "blend" of zenburn, but maybe in the future I'll have another stab at migrating my changes upstream.

Regarding `avy-background`. When I used ace-jump I did like the gray out stuff, mostly because it looked neat. Trying it without, is not as neat or pretty, but for me much more functional, as I used to easily get lost when the gray-out during search. To each his own.
